moveit_ros/planning_interface/moveit_cpp/include/moveit/moveit_cpp/planning_component.h

```diff
@@ -133,21 +163,24 @@ class PlanningComponent
 
   /** \brief Run a plan from start or current state to fulfill the last goal constraints provided by setGoal() using
    * default parameters. */
-  bool plan();
+  MoveItErrorCode plan();
```

Comment:
Didn't we discuss that `plan()` should return a `PlanSolution` that contains a `MoveItErrorCode` and that `PlanSolution` should override the `bool()` operator?